packages/server/utils/sql.util.ts

```diff
@@ -0,0 +1,13 @@
+export const toSQL = (query) => {
+  const { sql, bindings } = query.toSQL()
+  const fullQuery = bindings.reduce(
+    (acc, binding) =>
+      acc.replace(
+        '?',
+        binding instanceof Date ? `'${binding.toISOString()}'` : `'${binding}'`,
+      ),
+    sql,
+  )
+  console.log(fullQuery)
```

Comment:
Console.log

Reply:
Goal of the function is to print to console